packages/E2ETest/just-task.js

```diff
@@ -5,37 +5,17 @@
  * @ts-check
  */
 
-const {
-  task,
-  series,
-  argv,
-  tscTask,
-  eslintTask,
-  cleanTask,
-} = require('just-scripts');
+const { task, series, eslintTask, tscTask } = require('just-scripts');
 const fs = require('fs');
 
-task('clean', cleanTask({ paths: ['dist'] }));
 task('eslint', eslintTask());
 task('eslint:fix', eslintTask({ fix: true }));
-task('ts', () => {
-  return tscTask({
-    pretty: true,
-    ...(argv().production && {
-      inlineSources: true,
-    }),
-    target: 'es6',
-    module: 'commonjs',
-  });
-});
-
 task('prepareBundle', () => {
   const file = 'windows/ReactUWPTestApp/Bundle';
   if (!fs.existsSync(file)) {
     fs.mkdirSync(file);
   }
 });
 
-task('build', series('ts'));
-task('lint', series('eslint'));
+task('lint', series('eslint', tscTask()));
```

Comment:
>tscTask [](start = 30, length = 7)

Are we sure that our tsc tasks don't need to be in project order? i.e. we don't have any tsc task whose dependencies need dependent projects? 
At least if we make it part of 'lint' we should disable outputting files or at least update the output folder to be a temp folder so

Reply:
I set tsconfig to noEmit to not output files. 

our TS source doesn't rely on any other local projects, so I think we're fine.